pkgs/applications/networking/flent/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub, matplotlib, netperf, procps, pyqt5 }:
+
+buildPythonApplication rec {
+  name = "flent-${version}";
+  version = "1.2.2";
```

Comment:
pname="flent"
version=...
are enough, buildPythonApplication will generate the name

Reply:
ah ok, yeah I was originally doing a version from a git commit.